zne/noise_amplification/folding_amplifier/glorious_folding_amplifier.py

```diff
@@ -41,17 +70,17 @@ def _compute_folding_nums(self, noise_factor: float, num_nodes: int) -> Folding:
             Folding: named tuple containing full foldings, number of gates to partially fold and
             effective noise factor of the operation
         """
-        noise_factor = self._validate_noise_factor(noise_factor)
 
         foldings = (noise_factor - 1) / 2
-        full = int(foldings)
-        partial_foldings = foldings - full
-        partial = self._compute_best_estimate(num_nodes, partial_foldings)
+        full_foldings = int(foldings)
+        partial_foldings = foldings - full_foldings
+
+        gates_to_partial_fold = self._compute_best_estimated_gates(num_nodes, partial_foldings)
```

Comment:
Change to num_partial_folds

Reply:
I have had an idea on how to make this even cleaner, let's discuss on Tuesday! 🙂 It is based on computing the total number of foldings to perform in the whole circuit directly, and choosing how that gets distributed later